example/quantization/README.md

```diff
@@ -218,7 +218,7 @@ data/
 |---val.lxt
 |---val.idx
 model/
-|---ssd_vgg16_reduced_300.params
+|---ssd_vgg16_reduced_300-0000.params
```

Comment:
please add the note, user needs to rename, such as ssd-val-fc19a535.idx to val.idx.

Reply:
Another suggestion is to move/copy the SSD quantization instruction into ssd/README too.